src/System.Memory/tests/Span/Overflow.cs

```diff
@@ -60,6 +60,157 @@ public static void IndexOverflow()
             }
         }
 
+        [Fact]
+        [OuterLoop()]
+        public static void SliceStartInt32Overflow()
```

Comment:
Please use the same technique to allocate and use large memory as the other tests (i.e. use AllocationHelper, and skip on 32-bit platforms).

See examples here:
https://github.com/dotnet/corefx/blob/eb4bb7ff28d639dcd9fd0c505d78c6196a569f57/src/System.Memory/tests/Span/Clear.cs#L255-L300
https://github.com/rossslaney/corefx/blob/8b947b8f25498738e0b47314b2793827568865ec/src/System.Memory/tests/Span/Overflow.cs#L19-L61

Also, please make these specific to Windows/Mac only (as the exception may not get triggered on Linux).

Reply:
Please feel free to ask questions if you have any or if you get stuck.